src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs

```diff
@@ -561,44 +553,36 @@ private static string GetCommandId(Dictionary<string, long> dic)
 
             foreach (var kvp in indexes)
             {
-                TimeSpan difference;
-                if (lowestDatabaseUpTime > kvp.Value)
-                    difference = kvp.Value;
-                else
-                {
-                    difference = kvp.Value - newestIndexQueryTime;
-                    if (difference == TimeSpan.Zero && lowestDatabaseUpTime > kvp.Value)
-                        difference = kvp.Value;
-                }
+                var smallestTimeElapsedInCluster = kvp.Value;
```

Comment:
How do we ensure that it's really the smallest. It's not obvious so please add a comment.